src/Ledger/Conway/Conformance/Ratify.lagda

```diff
@@ -272,18 +274,35 @@ actualVotes Γ pparams cc ga votes
     λ where
 \end{code}
 \begin{code}
-      (true , just (just c'))  → just c'
-      _                        → nothing -- expired, no hot key or resigned
-
-  actualCCVote : Credential → Epoch → Vote
-  actualCCVote c e = case getCCHotCred (c , e) of
+    (true , just (just c'))  → just c'
+    _                        → nothing -- expired, no hot key or resigned
 \end{code}
 \begin{code}[hide]
-    λ where
+  getPoolParams : Credential → Maybe PoolParams
+  getPoolParams c = case lookupᵐ? stakeDelegs c of λ where  nothing → nothing
+                                                            (just kh)  → lookupᵐ? pools kh
 \end{code}
 \begin{code}
-      (just c')  → maybe id Vote.no (lookupᵐ? votes (CC , c'))
-      _          → Vote.abstain
+
+  SPODefaultVote : GovAction → VDeleg → Vote
+  SPODefaultVote (TriggerHF _) _ = Vote.no
+  SPODefaultVote NoConfidence (credVoter SPO c) = case getPoolParams c of λ where
+    nothing → Vote.no
+    (just p) → case lookupᵐ? delegatees (PoolParams.rewardAddr p) of λ where
+      (just noConfidenceRep)  → Vote.yes
+      (just abstainRep)       → Vote.abstain
+      _                       → Vote.no
+  SPODefaultVote _ (credVoter SPO c) = case getPoolParams c of λ where
+    nothing → Vote.no
+    (just p) → case lookupᵐ? delegatees (PoolParams.rewardAddr p) of λ where
+      (just abstainRep)  → Vote.abstain
+      _                  → Vote.no
+  SPODefaultVote _ _ = Vote.no
+
+  actualCCVote : Credential → Epoch → Vote
+  actualCCVote c e = case getCCHotCred (c , e) of λ where
```

Comment:
Why did you un-hide the `λ where`? It just makes the PDF more confusing to people who don't know Agda.

Reply:
For some reason, the only way I could get the TeX to compile without the "missing parameter l" error was to expose the `λ where` here and above.  I tried many permutations of exposing and hiding various parts of the code, and the one I ended up with was the closest to what we want while still getting the pdf to compile without error.

Reply:
It's probably just this bug again: https://github.com/agda/agda/issues/7290

You probably just need to indent the clauses of the `λ where` a bit more.

Reply:
Yes, it's exactly that bug.  I tried indenting all sorts of ways, but I'll try again to get it to compile while hiding all the `λ where`s and I'll also fix the bug mentioned above, of course.